src/query_error.h

```diff
@@ -74,6 +74,9 @@ typedef enum {
 typedef struct QueryError {
   QueryErrorCode code;
   char *detail;
+
+  // warnings
+  bool reached_maxprefixexpansions;
```

Comment:
This seems to break our naming convention, no?
Do we want underscores or camel case?